sites/h1/h1-en.md

```diff
@@ -0,0 +1,94 @@
+---
+{
+	"altLangPage": "h1-fr.html",
+	"breadcrumbs": [
+		{
+			"title": "GCWeb home",
+			"link": "https://wet-boew.github.io/GCWeb/index-en.html" },
+		],
+	"dateModified": "2023-06-19",
+	"description": "Documentation on the default H1 with short bold red underline.",
+	"language": "en",
+	"title": "Canada.ca H1"
+}
+---
+<div class="wb-prettify all-pre hide"></div>
+
+<dl class="horizontal">
+	<dt>Status</dt>
+	<dd>Stable</dd>
+	<dt>Version</dt>
+	<dd>1.0.0</dd>
+	<dt>Type</dt>
+	<dd>Canada.ca site style</dd>
+</dl>
+
+<h2>Introduction</h2>
+
+<h3>Purpose</h3>
+<p>
+	The purpose of the default H1 with short bold red underline indicates the main heading of the page.
+</p>
+
+<h3>Evaluation and report</h3>
+<p>There is no evaluation and report available for this component.</p>
+
+<h3>Guidance</h3>
+<ul>
+	<li><a href="https://design.canada.ca/styles/typography.html">Typography</a></li>
+</ul>
+
+<h3>Variants</h3>
+<p>This component supports only 1 variant:</p>
+<ul>
+	<li><a href="#variantDefault">H1 (default)</a></li>
+</ul>
```

Comment:
@delisma as we discussed the other day, I renamed this "component variant" concept for "Change set"